MeetupPlatformApi/DataTransferObjects/MeetupInputDto.cs

```diff
@@ -1,12 +1,18 @@
-﻿namespace MeetupPlatformApi.DataTransferObjects;
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace MeetupPlatformApi.DataTransferObjects;
 
 public class MeetupInputDto
 {
+    [Required]
     public string Name { get; set; }
 
+    [Required]
     public DateTime StartTime { get; set; }
 
+    [Required]
     public DateTime EndTime { get; set; }
 
+    [Required]
```

Comment:
Please, add `[MaxLength]` validation for this field. Max length should be somewhat around 4000 (almost 50 lines of text, if we assume, that line contains 80 characters).